### PR TITLE
updated documentation for plot_categorical

### LINKED ISSUE #16: write categorical plot function specifications and documentation

### PR DESCRIPTION
**Pull request by rbouwer**
Ready for review!

Tried to remain consistent with @phchen5 and @arturoboquin 's existing functions (and in keeping with some comments I made on those PR's https://github.com/UBC-MDS/PyXplor/pull/15/files & https://github.com/UBC-MDS/PyXplor/pull/13/files)

Let me know what you guys think (and will update based on 2pm discussion)

### CODE REVIEW

File: src/pyxplor/plot_categorical.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+
+def plot_categorical(
+        input_df: pd.DataFrame,
+        list_of_variables: list,
+        orientation: str='v',
```
Comment (rbouwer) [inline src/pyxplor/plot_categorical.py]:
```suggestion
```

File: src/pyxplor/plot_categorical.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        fig_width: int=10,
```
Comment (rbouwer) [inline src/pyxplor/plot_categorical.py]:
```suggestion
```

File: src/pyxplor/plot_categorical.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        fig_height: int=6,
```
Comment (rbouwer) [inline src/pyxplor/plot_categorical.py]:
```suggestion
        figsize: tuple = (10, 6),
```

File: src/pyxplor/plot_categorical.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        Height of the figure. Default is 6.
+
+    label_y_offset : int
+        Y-axis offset for label positioning. Default is 0.
+
+    label_fontsize : int
+        Font size for axis labels. Default is 10.
+
+    output_path : str, optional
+        Path to save the plot. Path to save the plot. Defaults to None, which means the plot will not be saved.
```
Comment (rbouwer) [inline src/pyxplor/plot_categorical.py]: Change to what @phchen5 wrote

File: src/pyxplor/plot_categorical.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
```
Comment (rbouwer) [inline src/pyxplor/plot_categorical.py]:
```suggestion
        Path to save the plot. Defaults to the current working directory.
```

**Review by arturoboquin — APPROVED** [review]

File: src/pyxplor/plot_categorical.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    Parameters
+    ----------
+    input_df : pd.DataFrame
+        The DataFrame that contains the categorical variables.
+
+    list_of_variables : list
+        List of categorical variables (column names) to be plotted.
+
+    orientation : {'v', 'h'}
+        Orientation of the plots, either vertical or horizontal. Default is is horizontal.
```
Comment (phchen5) [inline src/pyxplor/plot_categorical.py]: I think you forgot to remove this.
```suggestion
```

**Review by phchen5 — COMMENTED** [review]
Just remove orientation from the docstring and I think you're good to go!

File: src/pyxplor/plot_categorical.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+    fig_width : int
+        Width of the figure. Default is 10.
+
+    fig_height : int
+        Height of the figure. Default is 6.
```
Comment (phchen5) [inline src/pyxplor/plot_categorical.py]:
```suggestion
    figsize : tuple[int, int]
        The width and height of the figure size in a tuple.
```

File: src/pyxplor/plot_categorical.py [hunk 8]
```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+
+def plot_categorical(
+        input_df: pd.DataFrame,
+        list_of_variables: list,
+        figsize: tuple = (10, 6),
```
Comment (phchen5) [inline src/pyxplor/plot_categorical.py:6]: There might also be an issue putting an argument with default values in front of those without. So move this to the end of the argument list.

Comment (rbouwer) [reply src/pyxplor/plot_categorical.py:6]: I think everything below has a default - so hopefully should be good!

**Pull request merged**